packages/examples/examples/typescript/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "typescript-snap",
+  "version": "1.0.0",
+  "description": "The 'Hello, world!' of MetaMask Snaps, now written in TypeScript.",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/MetaMask/snap-template.git"
+  },
+  "license": "MIT",
+  "main": "src/index.ts",
+  "files": [
+    "dist/",
+    "images/",
+    "snap.manifest.json"
+  ],
+  "scripts": {
+    "build:clean": "yarn clean && yarn build",
+    "build:website": "node ./scripts/build-website.js",
+    "build": "mm-snap build --src ./src/index.ts",
```

Comment:
I'm not sure if there's an easy way to prevent having to add `--src` here? Ideally `mm-snap` would look at the `main` entry point in `package.json` unless `--src` is specified, but it seems that we hard code `src/index.js` everywhere right now.